aspnet-core/xml/ns-Microsoft.AspNetCore.Server.Kestrel.Core.Features.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Server.Kestrel.Core.Features">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides types that are used to describe feature components in the Kestrel web server.</summary>
+    <remarks>For more information, see <see href="https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.server.kestrel.core.features">Microsoft.AspNetCore.Server.Kestrel.Core.Features Namespace</see>.</remarks>
```

Comment:
I believe this is just linking to the page you're already on. I think you should either remove these namespace links, or change them to link to https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel.

Reply:
~/en-us~
https://docs.microsoft.com/aspnet/core/fundamentals/servers/kestrel

Reply:
Updated everywhere